package.json

```diff
@@ -17,6 +17,7 @@
     ]
   },
   "dependencies": {
+    "@react-native-community/datetimepicker": "^3.0.9",
```

Comment:
so you solved the issue with 3.0.9?

Reply:
I may have described this incorrectly, it's `expo install` that installs 3.0.4 -- and 3.0.4 doesn't work:
![image](https://user-images.githubusercontent.com/3383704/109967810-e0f9d800-7cbf-11eb-870c-e23385f2bf68.png)

...I was trying a variety of datepicker packages, each w/ both expo and yarn installs.

Reply:
github README documentation appears to encourage using expo install:

![image](https://user-images.githubusercontent.com/3383704/109969338-c7599000-7cc1-11eb-9af4-d77155bdf107.png)